src/map/clif.cpp

```diff
@@ -12557,7 +12557,7 @@ void clif_parse_skill_toid( struct map_session_data* sd, uint16 skill_id, uint16
 	sd->skillitem = sd->skillitemlv = 0;
 
 	if( SKILL_CHK_GUILD(skill_id) ) {
-		if( sd->state.gmaster_flag )
+		if( sd->state.gmaster_flag || skill_id == GD_CHARGESHOUT_BEATING )
```

Comment:
Guild members still unable to use the skill after this commit.

Reply:
You missed the commented out line then. But I just did more tests and changed some more behavior.

Reply:
Yeah I tested it before 06f5fed

After the latest commit, everything works fine perfectly.